### PR TITLE
Flashcard template

### PR DESCRIPTION
**Pull request by on-laf**
Sorted out the complicated (for me) routes in our views, and the foreign_key values

### CODE REVIEW

File: app/controllers/flashcard_templates_controller.rb [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  def index
+    @flashcard_templates = policy_scope(FlashcardTemplate)
+    @flashcard_template = FlashcardTemplate.new
+    authorize @flashcard_template
+  end
+
+  def new
+    @flashcard_template = FlashcardTemplate.new
+    authorize @flashcard_template
+    @topics = Topic.all.plug
```
Comment (arieivs) [inline app/controllers/flashcard_templates_controller.rb:13]: forgot to remove .plug

File: app/controllers/flashcard_templates_controller.rb [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  end
+
+  def create
+    @flashcard_template = FlashcardTemplate.new(flashcard_template_params)
+    @flashcard_template.teacher = current_teacher
+    authorize @flashcard_template
+    @flashcard_set = FlashcardSet.find(params[:flashcard_set_id])
+    @flashcard_template.flashcard_set = @flashcard_set
```
Comment (arieivs) [inline app/controllers/flashcard_templates_controller.rb:21]: I'd just put the authorize after asigning the set

Comment (on-laf) [reply app/controllers/flashcard_templates_controller.rb:21]: Why?

File: app/controllers/flashcard_templates_controller.rb (hunk 1 above)
Comment (on-laf) [reply app/controllers/flashcard_templates_controller.rb:13]: Done.

**Pull request merged**